### DISCUSSION TITLE
vue2中尝试使用vue-cli提供的快速原型开发来测试单个组件，但是全局安装了@vue/cli-service-global之后，使用vue serve 组件.vue命令之后，控制台显示已经在服务器上成功运行，但是界面什么都没有渲染出来

### DISCUSSION THREAD

**Discussion by Bowujun** [vuejs/vue-cli#7427] [Q&A]
<img width="1079" alt="image" src="https://github.com/vuejs/vue-cli/assets/133957391/6ec1a925-1504-4c90-82aa-f6e4bbf8a399">
<img width="1033" alt="image" src="https://github.com/vuejs/vue-cli/assets/133957391/8da99b5d-4048-4fe2-99eb-fa6b8d2b4448">

**Comment by monicaxun** [comment]
Me too. How can I resolve it.

**Reply by Bowujun** [reply]
reduce vue/cli version to 4.x
